dev/tasks/tasks.yml

```diff
@@ -1191,7 +1192,7 @@ tasks:
         UBUNTU: 22.04
       image: ubuntu-cpp-emscripten
 
-{% for python_version in ["3.8", "3.9", "3.10", "3.11", "3.12"] %}
+{% for python_version in ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"] %}
   test-conda-python-{{ python_version }}:
```

Comment:
I assume a conda build testing 3.13 will not yet work since conda-forge does not yet have Python 3.13. So maybe this change can be left out for now.

Also for a free-threaded variant, setting this up would not just work like this because then we need to install python from a custom channel? (https://py-free-threading.github.io/installing_cpython/#conda) But even then I am not sure if all other dependencies would then already have packages compatible with that python version?

Reply:
That's true. I removed 3.13 for now. Maybe it can be added later, when `conda-forge` does have a 3.13 version.